pyproject.toml

```diff
@@ -136,7 +136,6 @@ include-package-data = false
 
 [tool.setuptools.packages.find]
 where = ["src"]
-include = ["pymatgen", "pymatgen.*"]
```

Comment:
Perhaps just get rid of `include` altogether to use auto discover. Is there any sub-package we want to exclude?